test/fuse/linux/read_test.cc

```diff
@@ -143,87 +156,344 @@ TEST_F(ReadTest, ReadPartial) {
 
   // Check the 1-byte read request.
   GetServerActualRequest(iov_in);
-  EXPECT_EQ(in_header.len, sizeof(in_header) + sizeof(in_payload));
-  EXPECT_EQ(in_header.opcode, FUSE_READ);
-  EXPECT_EQ(in_payload.offset, 0);
+  EXPECT_EQ(in_header_read.len,
+            sizeof(in_header_read) + sizeof(in_payload_read));
+  EXPECT_EQ(in_header_read.opcode, FUSE_READ);
+  EXPECT_EQ(in_payload_read.offset, 0);
```

Comment:
Do we need to check `in_payload_read.fh` and `in_payload_read.flags`?

Reply:
There is nothing to check. fh is hard-coded in our fake inode. No need to check what we hard-coded in fake inode. There is no way and no reason to control flags. It's both limited by kernfs and out limited implementation.

Reply:
But you do set those attributes in line 63 of read_write.go. Can we still pass test if we erase line 63?